qadence/engines/torch/differentiable_expectation.py

```diff
@@ -6,7 +6,7 @@
 from typing import Any, Callable, Sequence
 
 import torch
-from pyqtorch.adjoint import AdjointExpectation
+from pyqtorch.differentiation import AdjointExpectation
```

Comment:
I was wondering why this was already in pyq 1.4.1: this was the change I accidentally committed to main 😬

Reply:
Ah np. I had to make a small PyQ release because I harmonised a bit the infra (test scripts, actions etc).